tutorials/CMakeLists.txt

```diff
@@ -276,11 +275,11 @@ endforeach()
 #--dependency for TMVA tutorials
 set (tmva-TMVAClassificationApplication-depends tutorial-tmva-TMVAClassification)
 set (tmva-TMVAClassificationCategory-depends tutorial-tmva-TMVAClassification)
-set (tmva-TMVAClassificationCategoryApplication-depends tutorial-tmva-TMVAClassificationCategory tutorial-tmva-TMVAClassificationCategory)
+set (tmva-TMVAClassificationCategoryApplication-depends tutorial-tmva-TMVAClassificationCategory tutorial-tmva-TMVAClassificationApplication)
 set (tmva-TMVAMulticlass-depends tutorial-tmva-TMVAMultipleBackgroundExample tutorial-tmva-TMVAClassificationCategory)
-set (tmva-TMVAMulticlassApplication-depends tutorial-tmva-TMVAMulticlass)
-set (tmva-TMVARegressionApplication-depends tutorial-tmva-TMVARegression)
-set (tmva-TMVARegression-depends tutorial-tmva-TMVAMulticlass-depends)
+set (tmva-TMVAMulticlassApplication-depends tutorial-tmva-TMVAMulticlass tmva-TMVAClassificationCategoryApplication)
```

Comment:
The dependent tests need to be specified with the full name: `tuturial-tmva-TMVAClassificationCategoryApplication`  instead of `tmva-TMVAClassificationCategoryApplication`